src/utils/eventDispatcher.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable max-classes-per-file */
+class Event {
```

Comment:
It would be nice if the utility functions had test code.

Reply:
Good😀  I will add testcases

Reply:
@ppeeou It's okay to start by opening a new issue.

Reply:
I have registered [a new issue #91](https://github.com/yorkie-team/yorkie-codepair/issues/91).

Reply:
@ppeeou Let's write test codes together! ✍️

Reply:
> @ppeeou It's okay to start by opening a new issue.

@hackerwins 
That's good too.
I can work on the weekend🤪


> @ppeeou Let's write test codes together! ✍️

@mojosoeun welcome 😄